### PR TITLE
[DNM] scale testing

### PR DESCRIPTION
**Pull request by sandreim**

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2344084

File: node/network/approval-distribution/src/tests.rs [hunk 1]
```diff
@@ -50,7 +51,8 @@ fn test_harness<T: Future<Output = VirtualOverseer>>(
 	let pool = sp_core::testing::TaskExecutor::new();
 	let (context, virtual_overseer) = test_helpers::make_subsystem_context(pool.clone());
 
-	let subsystem = ApprovalDistribution::new(Default::default());
+	let keysytore = test_helpers::mock::make_ferdie_keystore();
+	let subsystem = ApprovalDistribution::new(Default::default(), keysytore);
```
Comment (sergejparity) [inline node/network/approval-distribution/src/tests.rs:55]: Of course this is WIP. Just noticed strange var name "keysytore". Is it not by chance?

File: node/network/approval-distribution/src/tests.rs [hunk 2]
```diff
@@ -2283,7 +2285,8 @@ fn batch_test_round(message_count: usize) {
 	let mut state = State::default();
 
 	let (mut context, mut virtual_overseer) = test_helpers::make_subsystem_context(pool.clone());
-	let subsystem = ApprovalDistribution::new(Default::default());
+	let keysytore = test_helpers::mock::make_ferdie_keystore();
+	let subsystem = ApprovalDistribution::new(Default::default(), keysytore);
```
Comment (sergejparity) [inline node/network/approval-distribution/src/tests.rs:2289]: Of course this is WIP. Just noticed strange var name "keysytore". Is it not by chance?

File: node/network/approval-distribution/src/tests.rs (hunk 1 above)
Comment (sandreim) [reply node/network/approval-distribution/src/tests.rs:55]: nice catch, lol, typo. This is throw away changes mostly, just for testing purposes.

**Pull request closed without merging**